.github/ISSUE_TEMPLATE.md

```diff
@@ -11,8 +11,13 @@ Output from `nomad version`
 
 ### Reproduction steps
 
-### Nomad Server logs (if appropriate)
+### Job file (if appropriate)
 
 ### Nomad Client logs (if appropriate)
+If possible please post relevant logs in the issue.
 
-### Job file (if appropriate)
+Logs and other artifacts may also be sent to: nomad-oss-debug@hashicorp.com
```

Comment:
Add something like "When emailing logs, add a link to the relevant GH issue or PR".

Reply:
+1 - Potentially get them to add it to the subject, e.g `GH-2143: ....`